apps/retrospective/services/task.go

```diff
@@ -66,6 +66,50 @@ func (service TaskService) Get(id string, retroID string, sprintID string) (task
 	return &tasks[0], http.StatusOK, nil
 }
 
+// MarkDone ...
+func (service TaskService) MarkDone(id string, retroID string, sprintID string) (task *retroSerializers.Task, status int, err error) {
```

Comment:
let's use taskId as param for instance, we are using `sprintID` in sprint service

Reply:
Done